graphql_compiler/compiler/ir_lowering_sql/constants.py

```diff
@@ -20,13 +20,47 @@
 
 SUPPORTED_BLOCK_TYPES = (
     blocks.QueryRoot,
+    blocks.Filter,
 )
 
 SUPPORTED_OUTPUT_EXPRESSION_TYPES = (
     expressions.OutputContextField,
 )
 
 
+class Operator(object):
+    def __init__(self, name, cardinality):
+        """Represent an operator and it's underlying method."""
+        self.name = name
+        self.cardinality = cardinality
+
+
+class Cardinality(object):
```

Comment:
Any particular reason these should be attached to a class object, instead of being constants like `CARDINALITY_UNARY = 1`, `CARDINALITY_BINARY = 2` etc.?

Reply:
No they don't have to be, I can use that style.